docs/data-sources/identity_project_v3.md

```diff
@@ -14,6 +14,16 @@ data "opentelekomcloud_identity_project_v3" "project_1" {
 }
 ```
 
+### Query Current Project details
+
+If `name` or `domain_id` are not provided, data source gets info about current project.
```

Comment:
It works only if both are not provided? Maybe it should be ``` `name` and `domain_id` ``` ?

Reply:
It works when name or domain_id are not provided. Not only.